dcos/client.go

```diff
@@ -46,6 +46,8 @@ type APIClient struct {
 
 	Cosmos *CosmosApiService
 
+	Edgelb *EdgelbApiService
```

Comment:
nit: perhaps we should rename the variables to use `API` instead of `Api` as done for the `APIClient` type. Not related to this PR.

Reply:
I'll look into it, for now I've updated the properties to be eg. `Cosmos` instead of `CosmosApi`